tools/facepalm/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:14.04
+FROM postgres:9.5
```

Comment:
If prod is stuck using postgres 9.2, we probably should stick with that version in dev as well.